.github/workflows/update-versions.yml

```diff
@@ -20,12 +20,11 @@ jobs:
         id: get-version
         run: |
           # Get all releases and find the first one starting with 'v'
-          RELEASE_INFO=$(curl -s "https://api.github.com/repos/filecoin-project/lotus/releases" | \
-            jq -r '[.[] | select(.tag_name | startswith("v"))] | first')
```

Comment:
I would still make sure that the `latest` tag isn't a `miner/vX.X.X` release by leaving this tag_name check in.